langchain/src/vectorstores/cassandra.ts

```diff
@@ -10,12 +10,20 @@ export interface Column {
   name: string;
 }
 
+export interface Index {
+  name: string;
+  value: string;
+}
+
+export type CassandraVectorStoreFilterType = string[];
```

Comment:
You will need to `declare` this in your class - see:

https://github.com/langchain-ai/langchainjs/blob/main/langchain/src/vectorstores/analyticdb.ts#L45